packages/quantic/cypress/e2e/default-2/search-box/search-box-expectations.ts

```diff
@@ -0,0 +1,146 @@
+      expectedQuery: string,
+      expectedRecentQueriesInLocalStorage?: {LSkey: string; queries: string[]}
+    ) => {
+      cy.wait(InterceptAliases.Search)
+        .then((interception) => {
+          const requestBody = interception.request.body;
+          expect(requestBody).to.have.property('q', expectedQuery);
+          if (expectedRecentQueriesInLocalStorage) {
+            const {LSkey, queries} = expectedRecentQueriesInLocalStorage;
+            const recentQueries = window.localStorage.getItem(LSkey);
+            expect(recentQueries).to.equal(JSON.stringify(queries));
+          }
+        })
+        .logDetail(
+          `should make a search request with the query ${expectedQuery}`
+        );
+    },
+    displaySearchBoxInput: (display: boolean, textarea = false) => {
+      selector
+        .input(textarea)
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(`${should(display)} display the input search box`);
+    },
+    displaySearchButton: (display: boolean) => {
+      selector
+        .searchButton()
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(`${should(display)} display the search button`);
+    },
+    displaySuggestionList: (display: boolean) => {
+      selector
+        .suggestionList()
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(`${should(display)} display the query suggestions list`);
+    },
+    numberOfQuerySuggestions: (value: number) => {
+      selector
+        .querySuggestions()
+        .should('have.length', value)
+        .logDetail(`should display ${value} query suggestions`);
+    },
+    searchInputHasAriaOwnsValue: (textarea = false) => {
+      selector
+        .input(textarea)
+        .then((inputElement) => {
+          selector.suggestionList().then((suggestionListElement) => {
+            const actualValue = inputElement[0].getAttribute('aria-owns');
+            expect(actualValue).to.equal(suggestionListElement[0].id);
+          });
+          return;
+        })
+        .logDetail(
+          'The search input should have the id of the suggestion list as its value for the aria-owns attribute'
+        );
+    },
+    searchInputHasAriaActiveDescendantValue: (
+      textarea = false,
+      index: number
+    ) => {
+      selector
+        .input(textarea)
+        .then((inputElement) => {
+          selector
+            .querySuggestionByIndex(index)
+            .then((suggestionOptionElement) => {
+              const actualValue = inputElement[0].getAttribute(
+                'aria-activedescendant'
+              );
+              expect(actualValue).to.equal(suggestionOptionElement[0].id);
+            });
+          return;
+        })
+        .logDetail(
+          `the aria-activedescendant attribute of the searchbox input should have the id of the ${index}-th suggestion as its value`
+        );
+    },
+    searchInputHasNoAriaActiveDescendantValue: (textarea = false) => {
```

Comment:
Isn't this more of a unit test honnestly?
Do we really need a e2e test for this?